### ISSUE TITLE
Blastpgp. Obsolete Software 

### ISSUE THREAD

**Issue by GrimSmiler** [Rostlab/ConSurf#6]
Blastpgp is no longer available.

It is encouraged to use blast+ (`sudo apt-get install ncbi-blast+`)

**Comment by GrimSmiler** [comment]
Executable for linux can be downloaded [here](http://prodata.swmed.edu/download/pub/promals/promals/blastpgp/)

**Comment by GrimSmiler** [comment]
http://mirrors.vbi.vt.edu/mirrors/ftp.ncbi.nih.gov/blast/executables/release/2.2.26/

**Comment by GrimSmiler** [comment]
Managed to run consurf
returns an error:

```
ConSurf Version 1.0.6
====================================
Process Started at 20:55:11

Output files could be found in output directory: /vagrant/ConSurf-develop/output_test//
[blastpgp] ERROR: No argument value given for Database

run_blast : run of blast fail. /tmp/WZNObQWiVi//1lk2_A.protein_query.blast is zero or not exists
```

**Comment by juanmirocks** [comment]
This issue appears to be a duplicate of https://github.com/Rostlab/ConSurf/issues/8

**Comment by GrimSmiler** [comment]
issue found. fix will be presented in #8

**Comment by juanmirocks** [comment]
Great job
